### ISSUE TITLE
Changement de Nom dans HomeKit : Name change in HomeKit:

### ISSUE THREAD

**Issue by Stewart06** [nfarina/homebridge-tesla#113]
Bonjour,

Avant tout je tien a vous remercier pour ce fabuleux plugin qui me permet de gérer à distance le mode sentinelle ainsi que la climatisation à l'aide d'un simple bouton.

Depuis la dernière mise à jour 4.3.0, à chaque fois que je redémarre le Raspberry et donc le plugin, le nom donner dans HomeKit reviens au nom d'origine.

Il me semble que cela ne le faisait pas avant la dernière mise jour.

Pourriez-vous régler cela ?

Merci d'avance pour le travail que cela peut engendrer.

Good morning,

First of all I would like to thank you for this fabulous plugin which allows me to remotely manage the sentinel mode as well as the air conditioning using a simple button.

Since the last 4.3.0 update, each time I restart the Raspberry and therefore the plugin, the name given in HomeKit returns to the original name.

Seems to me it didn't before the last update.

Could you fix this?

Thank you in advance for the work that this may entail.

**Comment by nfarina** [comment]
Thanks for reporting this! It looks like supporting the ConfiguredName characteristic requires more work than I thought, so I've reverted that change and restored the previous behavior. If you install the latest version of the plugin (4.3.4) your custom names should stick again.

**Comment by Stewart06** [comment]
Je viens de mettre à jour le plugin, est maintenant des que je modifie le nom d'un accessoire explosé à HomeKit celui-ci redeviens comme à l'origine après quelques secondes. J'ai tout essayé mais cela ne change rien.

Concernant la fonction caractéristique ConfiguredName, le plugin SamsungTizen fonctionne très bien, si jamais cela peut vous aidez dans vos travaux.

I just updated the plugin, now as soon as I change the name of an exploded accessory to HomeKit it returns to the original after a few seconds. I tried everything but it didn't change anything.

Regarding the characteristic function ConfiguredName, the SamsungTizen plugin works very well, if it can help you in your work.

**Comment by nfarina** [comment]
I saw the same behavior with my car where the name returned to the original after a few seconds. I had to remove the accessory from Homebridge, restart, allow the Home app to see that it's gone, then add it back again. The Home app seems to "cache" the old presence of the ConfiguredName characteristic.

I would love to explore proper support for ConfiguredName, and will check out that plugin when I have time - sadly this is really a project I can only spare a few minutes on occasionally.
